addons/controllers/csi/vspherecsiconfig_controller.go

```diff
@@ -39,6 +40,39 @@ type VSphereCSIConfigReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+var providerServiceAccountRBACRules = []rbacv1.PolicyRule{
```

Comment:
nit(non-blocking): Was thinking what if we put this block inside of `vspherecpiconfig_utils.go` ? But defining it here is totally fine.

Reply:
I added it here because originally I wanted to create the clusterole as part of controller init but ran into issues determining if TKGS or not so the create is in the reconcile loop now.